components/Home.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import styled from 'styled-components/native';
+import { TouchableOpacity, ScrollView } from 'react-native';
+import { fetchDetail } from '../reducers/meal';
+import { meal } from '../reducers/meal';
+import { Linking } from 'react-native';
```

Comment:
I think this Linking should be included in line 4 with the other imports from react-native